src/Illuminate/Foundation/Application.php

```diff
@@ -1051,7 +1051,95 @@ public function setLocale($locale)
      */
     public function isLocale($locale)
     {
-        return $this->getLocale() == $locale;
+        return $this->getLocale() === $locale;
+    }
+
+    /**
+     * Get default application locale
+     *
+     * @return string
+     */
+    public function getDefaultLocale()
+    {
+        return config('app.fallback_locale');
+    }
+
+    /**
+     * Get additional locales supported by application
+     *
+     * @return array
+     */
+    public function getAdditionalLocales()
+    {
+        return config('app.additional_locales');
+    }
+
+    /**
+     * Get array of all locales supported by application
+     *
+     * @return array
+     */
+    public function getSupportedLocales()
+    {
+        $app_locales = config('app.additional_locales');
+        $app_locales[] = config('app.fallback_locale');
+        return $app_locales;
+    }
+
+    /**
+     * Get application configuration for default locale presence in URL
+     *
+     * @return bool
+     */
+    public function isDefaultLocaleInUrl()
+    {
+        return config('app.default_locale_in_url') === true;
```

Comment:
isDefaultLocaleInUrl() is used in other classes and I don't realy want to access config directly when it is related to App. Also it might be useful for 3rd party code.